app/views/shared/_navbars.html.haml

```diff
@@ -0,0 +1,114 @@
+          -#   %span.sr-only View notifications
+          -#   %svg.h-6.w-6{"aria-hidden" => "true", :fill => "none", :stroke => "currentColor", "stroke-width" => "1.5", :viewbox => "0 0 24 24"}
+          -#     %path{:d => "M14.857 17.082a23.848 23.848 0 005.454-1.31A8.967 8.967 0 0118 9.75v-.7V9A6 6 0 006 9v.75a8.967 8.967 0 01-2.312 6.022c1.733.64 3.56 1.085 5.455 1.31m5.714 0a24.255 24.255 0 01-5.714 0m5.714 0a3 3 0 11-5.714 0", "stroke-linecap" => "round", "stroke-linejoin" => "round"}
+          -# / Separator
+          -# .hidden.lg:block.lg:h-6.lg:w-px.lg:bg-gray-200{"aria-hidden" => "true"}
+          
+          / Profile dropdown
+          .relative{"x-data": "{ dropdownOpen: false }"}
+            %button#user-menu-button.flex.items-center{"aria-expanded" => "false", "aria-haspopup" => "true", :class => "-m-1.5 p-1.5", :type => "button", "@click": "dropdownOpen = !dropdownOpen"}
+              %span.sr-only Open user menu
+              %img.h-8.w-8.rounded-full.bg-gray-50{:alt => "", :src => "https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"}/
+              %span.hidden.lg:flex.lg:items-center
+                %span.ml-4.text-sm.font-semibold.leading-6.text-gray-900.dark:text-white{"aria-hidden" => "true"} Tom Cook
+                %svg.ml-2.h-5.w-5.text-gray-400{"aria-hidden" => "true", :fill => "currentColor", :viewbox => "0 0 20 20"}
+                  %path{"clip-rule" => "evenodd", :d => "M5.23 7.21a.75.75 0 011.06.02L10 11.168l3.71-3.938a.75.75 0 111.08 1.04l-4.25 4.5a.75.75 0 01-1.08 0l-4.25-4.5a.75.75 0 01.02-1.06z", "fill-rule" => "evenodd"}
+            /
+              Dropdown menu, show/hide based on menu state.
+            .absolute.right-0.z-10.w-32.origin-top-right.rounded-md.bg-white.dark:bg-card-dark.py-2.shadow-lg.ring-1.focus:outline-none.text-gray-900.dark:text-white{"aria-labelledby" => "user-menu-button", "aria-orientation" => "vertical", :class => "mt-2.5 ring-gray-900/5", :role => "menu", :tabindex => "-1", "x-show": "dropdownOpen", "@click.outside": "dropdownOpen = false", "x-transition:enter": "ease-out duration-100", "x-transition:enter-start": "transform opacity-0 scale-95", "x-transition:enter-end": "transform opacity-100 scale-100", "x-transition:leave": "transition ease-in duration-75", "x-transition:leave-start": "transform opacity-100 scale-100", "x-transition:leave-end": "transform opacity-0 scale-95"}
+              -# %a#user-menu-item-0.block.px-3.py-1.text-sm.leading-6.text-gray-900{:href => "#", :role => "menuitem", :tabindex => "-1"} Your profile
+              = link_to "Profile", edit_user_registration_path, class: "block px-3 py-1 text-sm leading-6 hover:bg-gray-50 dark:hover:bg-body-dark"
+              = link_to "Sign Out", destroy_user_session_path, method: :delete, class: "block px-3 py-1 text-sm leading-6 hover:bg-gray-50 dark:hover:bg-body-dark"
```

Comment:
@danielpaul can you have a look at this sign_out, when I click it I get "No route matches [GET] '/users/sign_out'". I checked the routes and the destroy_user_session path is there